Dockerfile

```diff
@@ -27,7 +27,7 @@ ARG BUILD_NUMBER
 RUN rm -rf target/release/.fingerprint/safe-client-gateway*
 RUN cargo build --release --locked
 
-# Image Stage
+# Runtime Image
 FROM debian:buster-slim
```

Comment:
No, because this is just what is used inside docker ... you can use a debian docker image on a m1 mac for all that matter there should be no complications if docker itself is properly working